### PR TITLE
Add staging config for webpacker

### PR DESCRIPTION
**Pull request by jfranciscosousa**

### CODE REVIEW

File: config/webpacker.yml [hunk 1]
```diff
@@ -59,6 +59,15 @@ test:
   # Compile test packs to a separate directory
   public_output_path: packs-test
 
+staging:
```
Comment (frm) [inline config/webpacker.yml]: Can't we use production environment settings? I thought the buildpack was already precompiling the assets in staging. It's supposed to be 100% similar to production as far as the infrastructure goes.

Comment (jfranciscosousa) [reply config/webpacker.yml]: It's the same, you need this though for it to work.

Comment (jfranciscosousa) [reply config/webpacker.yml]: Check it out now, I made it inherit from production.

Comment (frm) [reply config/webpacker.yml]: Oooh, I like this

**Review by frm — APPROVED** [review]

**Pull request merged**